### ISSUE TITLE
When using expanded style, the ampersand construction does not work with multiple selectors

### ISSUE THREAD

**Issue by Drakiri** [madskristensen/WebEssentials2013#1916]
When using expanded style, the ampersand construction does not work with multiple selectors:

.diva, .divb
{
    .divc
        {
        &:after
            {
         width: 10%;   
        }   
    }
}

becomes

when output style is 'Expanded' it is wrong:
.diva
**.divc
:after**
, .divb
.divc:after {
  width: 10%;
}

while with 'Compact' output style is fine:
.diva **.divc:after**, .divb .divc:after { width: 10%; }

**Comment by am11** [comment]
I couldn't figure out whether you are facing the issue with Less or Scss, so I tested with both Less (`test-1916.less`) and Scss (`test-1916.scss`) with the following source code using WE 2.6.12 on WE 2013 Update 5:

```scss
.diva, .divb {
    .divc {
        &:after {
            width: 10%;
        }
    }
}
```

`.css` output for `test-1946.scss`:

```css
.diva .divc:after, .divb .divc:after {
  width: 10%;
}
```

`.css` output for `test-1946.less`:

```css
.diva .divc:after,
.divb .divc:after {
  width: 10%;
}
```

`.min.css` output for `test-1946.scss`:

```css
.diva .divc:after,.divb .divc:after{width:10%}
```

`.min.css` output for `test-1946.less`:

```css
.diva .divc:after,.divb .divc:after{width:10%}
```

**Comment by Drakiri** [comment]
I'm using SCSS (should have mentioned that, in hindsight). 

The problem occurs when setting 'Tools -> Options -> Web Essentials -> SASS -> Output Style to Expanded.

The ouput of the example above becomes:

```css
.diva
.divc
:after
, .divb
.divc:after {
    width: 10%;
}
```

I'm using version 2.6.12 from 17th of july 2015.

**Comment by Drakiri** [comment]
Oh, and Visual Studio is still at Update 4, maybe this is causing the problem, will try to update to version 5 during break time and see if that helps.

**Comment by am11** [comment]
Update 5 link: https://www.visualstudio.com/downloads/download-visual-studio-vs#d-premium *(its for premium, choose the correct flavor. after download, rename `vs_<flavour-name>` to `vs_<flavour-name>.exe` as this seems like their deployment bug)*

I tested with the following settings:

![test1](https://cloud.githubusercontent.com/assets/3840695/8846224/d967761c-312d-11e5-9240-b659992cbc1e.PNG)

![test1-scss](https://cloud.githubusercontent.com/assets/3840695/8846218/cc4792dc-312d-11e5-907c-29a81279e5fa.PNG)

![test1-we](https://cloud.githubusercontent.com/assets/3840695/8846273/446c70d4-312e-11e5-9289-64b1180970b6.PNG)

**Comment by Drakiri** [comment]
I'm stumped :(

![image](https://cloud.githubusercontent.com/assets/13414201/8849764/97586bf4-3142-11e5-8493-d798d734fcc4.png)

![image](https://cloud.githubusercontent.com/assets/13414201/8849776/a61dfb68-3142-11e5-905c-683b6bb95641.png)

![image](https://cloud.githubusercontent.com/assets/13414201/8849797/d56a444e-3142-11e5-95ea-7cb214d6f674.png)

**Comment by Drakiri** [comment]
![image](https://cloud.githubusercontent.com/assets/13414201/8849965/57fe1268-3144-11e5-83bd-27f00381eaef.png)

![image](https://cloud.githubusercontent.com/assets/13414201/8849972/644f6850-3144-11e5-896a-a536323a267d.png)

**Comment by am11** [comment]
TLDR; (*workaround*) when the Scss file is opened in the VS Sass editor, press Ctrl+K,D to format the document and compile, it will produce the desired output.

---

I was going to report this issue to libsass, as it is happening in the current released version libsass (via node-sass), then I tested with sassc with latest libsass master and it seems to be fixed. 

@xzyfer, @mgreter, just FYI.

With node-sass v3.2.0 (=> libsass v3.2.5):

#### Case: Correctly Formatted Input:

Input:

```scss
.diva, .divb {
  .divc {
    &:after {
      width: 10%;
    }
  }
}
```

Output:

```css
.diva .divc:after, .divb .divc:after {
  width: 10%;
}
```

#### Case: Unformatted Input:

Input:

```scss
.diva, .divb
 {
    .divc
     {
        &:after
        {
            width: 10%;
        }
    }
}
```

Output:

```css
.diva
.divc
:after
, .divb
.divc:after {
  width: 10%;
}
```

**Comment by xzyfer** [comment]
As @am11 points out this is resolved in the latest LibSass master. We'll be shipping libsass and node-sass betas this month.

**Comment by Drakiri** [comment]
@am11 : we're working around it at the moment by setting web essentials output to Compact, as our VS SCSS output has Expanded Braces style (which means ctrl+k,d will produce exactly the wrong input for WE). 

We don't want to change the formatting of the SCSS file for source control reasons, so until a fixed version is released we'll make do with Compact compiled output, 

Thanks for all your help guys :)

**Comment by am11** [comment]
Thanks @xzyfer for the confirmation! :)

@Drakiri, there is another alternative: as WE packs both LibSass and Ruby-Sass, you can set `Use Ruby Runtime` to `True` in WE>Scss settings and it will compile the `Expanded` output correctly. (note: Ruby Sass compiler is much slower than libsass written in C++ :wink:)
